src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js

```diff
@@ -1,30 +1,142 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
+import moment from "moment";
 import PropTypes from "prop-types";
-import { Header, Footer, Button } from "shared-components";
+import { Header, Footer, Button, Table, Alert } from "shared-components";
+import { MdCancel } from "react-icons/md";
 import axios from "axios";
 import { propTypes } from "../../../types/propTypes";
 
 export default function PackageReview({ page: { header, packageId } }) {
+  const [error, setError] = useState(false);
+  const [packageNoDetails, setPackageNoDetails] = useState([
+    {
+      name: "Package Number:",
+      value: "",
+      isNameBold: false,
+      isValueBold: true,
+    },
+    { name: "Submitted By:", value: "", isNameBold: false, isValueBold: true },
+    {
+      name: "Submitted Date:",
+      value: "",
+      isNameBold: false,
+      isValueBold: true,
+    },
+  ]);
+  const [courtFileDetails, setCourtFileDetails] = useState([
+    {
+      name: "Court File Number:",
+      value: "",
+      isNameBold: false,
+      isValueBold: true,
+    },
+    { name: "Submitted To:", value: "", isNameBold: false, isValueBold: true },
+    {
+      name: "Filing Comments:",
+      value: "",
+      isNameBold: false,
+      isValueBold: true,
+    },
+  ]);
+
   useEffect(() => {
-    console.log(packageId);
     axios
       .get(`filingpackage/${packageId}`)
       .then((response) => {
-        console.log(response);
+        try {
+          const packageNo = response.data.packageNumber || "n/a";
+          let submittedBy = "n/a";
+          if (response.data.submittedBy.firstName) {
+            if (response.data.submittedBy.lastName) {
```

Comment:
Can we use &&?

Reply:
Another great question. I originally had &&, but `yarn lint` failed when I did.

Reply:
oh yeah, I remembered that. keep it this way then! those rules are a bit weird sometime ;-)